hack/install_cli_latest.sh

```diff
@@ -2,6 +2,7 @@
 
 set -e
 
+TMPDIR=${TMPDIR:-/tmp}
 cd ${TMPDIR}
```

Comment:
can we `cd` always or should we add `mkdir -p ${TMPDIR}` ?

Reply:
I decided to get rid of the `cd` command